jenkins/scripts/cbts/coverage_utils/sitecustomize.py

```
@@ -0,0 +1,206 @@
+    # cleanly via atexit, so we skip the periodic thread there.
+    _PERIODIC_SAVE_SECONDS = 5
+    _stop_event = threading.Event()
+
+    def _periodic_save():
+        # Event.wait is interruptible, unlike time.sleep, so atexit
+        # can signal a fast exit. Returns True when the event is set,
+        # which is our stop signal.
+        while not _stop_event.wait(_PERIODIC_SAVE_SECONDS):
+            try:
+                cov.save()
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)